tests/lib/utils.py

```diff
@@ -39,6 +39,46 @@ def cvp_login():
         return cvp_client
 
 
+def generate_test_ids_list(val):
```

Comment:
Is this needed ? This function is basically identity. If you do not pass `ids` kwarg to the pytest parametrize decorator, pytest will display the actual list element value as id, I suppose.

Reply:
It helps to get a name to the test compare to iteration number when it is not defined.

Reply:
Ok, function removed as unused in this context and `generate_test_ids_dict` is the only one to use